debug.log

```diff
@@ -0,0 +1,3 @@
+[1205/145409.417:ERROR:process_info.cc(359)] UncheckedAllocate
```

Comment:
Not sure what this file is - can you delete it from the commit please? :)

Reply:
On it rn. Thank you for pointing this out.